exchange/index.go

```diff
@@ -546,7 +547,7 @@ func (idx *Index) CleanBlockStore(ctx context.Context) error {
 		return err
 	}
 	for k := range kc {
-		key := cid.NewCidV1(cid.DagCBOR, k.Hash())
+		key := cid.NewCidV1(cid.Raw, k.Hash())
```

Comment:
You shouldn't need to do this since all the keys come back with a raw codec by default